src/System.Net.Requests/tests/FileWebRequestTest.cs

```diff
@@ -186,9 +186,13 @@ public abstract class FileWebRequestTestBase
             }
         }
 
+        protected virtual bool EnableConcurrentReadWriteTests => true;
+
         [Fact]
         public async Task ConcurrentReadWrite_ResponseBlocksThenGetsNullStream()
         {
+            if (!EnableConcurrentReadWriteTests) return;
```

Comment:
Kind of odd to have a test that does nothing.  Can we move this test method into the derived class(es) that *do* support concurrent tests?